src/sugar/index.ts

```diff
@@ -18,7 +18,7 @@ export function ensureClassicAddress(account: string): string {
     // must not have an embedded tag. In other words,
     // `tag` should be `false`.
     if (tag !== false) {
-      throw new Error(
+      throw new errors.XrplError(
```

Comment:
Why is this `error.XrplError` instead of just `XrplError`?

Reply:
Because we are we exporting errors from this file as well.

Reply:
We shouldn't be exporting errors here too